language-support/hs/bindings/test/DA/Ledger/Tests.hs

```diff
@@ -516,7 +516,7 @@ bucket = VRecord $ Record Nothing
     , RecordField "contract"$ VContract (ContractId "xxxxx")
     , RecordField "list"    $ VList []
     , RecordField "int"     $ VInt 42
-    , RecordField "decimal" $ VDecimal "123.456"
+    , RecordField "decimal" $ VDecimal (read "123.456")
```

Comment:
Does this work?
```suggestion
    , RecordField "decimal" $ VDecimal 123.456
```

Reply:
no, it didn't. - that why I added `read`
should it?
would it need a `FromString` instance for `Fixed` or something?

remember, I don't know the details. I just let the compiler errors guide me :)

Reply:
Note that @hurryabit proposed `VDecimal 123.456` not `VDecimal "123.456"`. That definitely does not need an `IsString` instance and if it doesn’t work, I’d be interested in the error.

Reply:
For my proposal we need a `Fractional` instance, which `Fixed E10` should have.

Reply:
AH, sorry I should have read your proposal more closely.
`VDecimal 123.456` works like a charm

From `VDecimal "123.456"`, we get the following error as expected:
```
language-support/hs/bindings/test/DA/Ledger/Tests.hs:519:40-48: error:
    * No instance for (Data.String.IsString (Data.Fixed.Fixed LF.E10))
        arising from the literal `"123.456"'
    * In the first argument of `VDecimal', namely `"123.456"'
      In the second argument of `($)', namely `VDecimal "123.456"'
      In the expression: RecordField "decimal" $ VDecimal "123.456"
    |
519 |     , RecordField "decimal" $ VDecimal "123.456"
    |                                        ^^^^^^^^^
```